app/helpers/application_helper.rb

```diff
@@ -37,7 +37,9 @@ def foi_error_messages_for(*params)
     options = params.last.is_a?(Hash) ? params.pop.symbolize_keys : {}
     objects = params.collect { |object_name| instance_variable_get("@#{object_name}") }.compact
     count   = objects.inject(0) { |sum, object| sum + object.errors.count }
-    unless count.zero?
+    if count.zero?
```

Comment:
Returning an empty string is no different to return nil. So we could use a guard clause here instead. 

```suggestion
    return if count.zero?
```

Reply:
Fixed in Use guard clause instead of if/else